documentation/endpoints/activity.md

```diff
@@ -21,6 +21,9 @@ GET /api/v2/activity
 - **user** _(integer)_
   - Whether or not to include information about the user with the response.
   - e.g. `/activity?include=user`
+- **updated_at** _(timestamp)_
+  - Return records that have been updated at after the updated_at param value. 
+  - e.g. `/activity?filter[updated_at]=2017-05-25 20:14:48`
```

Comment:
@katiecrane it does! 
![screen shot 2017-05-31 at 10 22 41 am](https://cloud.githubusercontent.com/assets/9019452/26636564/2b754104-45eb-11e7-8148-e07ca38fa8c7.png)